exporter/containerimage/attestations.go

```diff
@@ -20,8 +20,8 @@ import (
 	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
 	spdx_json "github.com/spdx/tools-golang/json"
-	"github.com/spdx/tools-golang/spdx/common"
-	spdx "github.com/spdx/tools-golang/spdx/v2_3"
+	"github.com/spdx/tools-golang/spdx/v2/common"
+	spdx "github.com/spdx/tools-golang/spdx/v2/v2_3"
```

Comment:
The versioning is really confusing in this package; I looked at this and thought "why are we still using `v1` as well, but this is not a v2 module, but a package named `v2` (and `v2_3` under it 😞 😂)